src/components/Checkbox/index.native.js

```diff
@@ -0,0 +1,17 @@
+import _ from 'lodash';
+import React from 'react';
+import BaseCheckbox from './BaseCheckbox';
+
+const Checkbox = props => (
+    <BaseCheckbox
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        {..._.omit(props, 'onMouseDown')}
+        onPress={props.onPress || props.onMouseDown}
```

Comment:
Should I add a commit with this comment?

Reply:
It just doesn't make much sense to me why native components would ever need onPress. Couldn't it be removed entirely?

Reply:
It is actually essential. You see what I'm doing here is not executing any code on the `onMouseDown` event, but if the `onMouseDown` prop is passed, then I am executing the prop passed on the `onPress` event because the `onMouseDown` doesn't fire in native.

Reply:
That makes sense to me for the general component, but there will never be an onMouseDown event for the native component, right? Or am I missunderstanding

Reply:
Yes there will never be an onMouseDown According to my knowledge. You are you proposing to remove the omit from the props spread?

Reply:
> At first I thought you were referring to the onPress={props.onPress || props.onMouseDown}, that line is essential

Oh yeah, sorry that is what I meant. It would be great to add a comment that explains why this is necessary --- to help people like me who are a bit confused by seeing it there.

Reply:
Every checkbox can have two click handlers `onPress` and `onMouseDown`. As there is no `onMouseDown` event on native, any prop that is passed to be executed on the `onMouseDown` event should be executed on the `onPress` event instead.

Reply:
Hey @Someone-here. This has made me realise that the solution is just a bit too complex for solving a small issue. Let's drop this one and move onto other bugs 👍